src/snowflake/connector/cursor.py

```diff
@@ -1524,3 +1506,31 @@ def __init__(self, connection) -> None:
             connection,
             use_dict_result=True,
         )
+
+
+def __getattr__(name):
```

Comment:
I don't see tests for this. Please at least manually test that this works as expected

Reply:
I have added test for this, it's working as expected